### PR TITLE
Upated some instructions

### PR DESCRIPTION
**Pull request by NollKollTroll**
Updated the instructions to fit better with my experience assembling the PCB.
I hope this PR is correctly done.

### CODE REVIEW

File: ASSEMBLY.md [hunk 1]
```diff
@@ -54,8 +57,15 @@ The general rule is to fit the parts with the lowest Z-height first (i.e. the fl
...
 * The Raspberry Pi Pico (if you want to solder it straight to the board) (U201)
 * Any 2.54mm pin headers and jumpers
-  * Don't worry about J906, J907 and J908 - they're just for debugging/testing/probing
-  * Ensure you leave out pin 8 on the J802 PC case audio connector, as most cases have a blank position on the mating connector to make sure you can't put it in backwards. Pin 8 is the top row, one in from the right; if you look carefully it's the only pad on that 10-way header that doesn't have a PCB trace running to it.
+  * Don't worry about J906, J907 and J908 - they're just for debugging/testing/probing. It's easier to put an oscilloscope probe directly into a hole.
+  * Ensure you leave out pin 8 on the J802 PC case audio connector, as most cases have a blank position on the mating connector to make sure you can't put it in backwards.<p>
+```
+.    +----+----+----+----+----+
+.    |  2 |  3 |  6 |  X | 10 |
+Case +----+----+----+----+----+
```
Comment (thejpster) [inline ASSEMBLY.md:65]: What does the word *Case* mean or imply here?

Comment (NollKollTroll) [reply ASSEMBLY.md:65]: Case is printed on the PCB. I thought it might make the orientation more certain.

Comment (thejpster) [reply ASSEMBLY.md:65]: Oh of course it is!

**Comment by thejpster** [comment]
There's another picture of the audio header in the section *Not using a PCB Case*. Could we make them look similar (albeit one has the jumpers indicated)?

**Comment by thejpster** [comment]
Maybe also add a note "See *Not using a PCB Case* if you don't actually connect this header to your ATX case, as you'll need to fit some jumpers". Or similar words.

**Comment by NollKollTroll** [comment]
Will do, great addition!

**Comment by NollKollTroll** [comment]
Updated regarding jumpers.

**Review by thejpster — APPROVED** [review]

**Comment by thejpster** [comment]
Thank you! :)

**Pull request merged**